alacritty_terminal/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 0.23.0-dev
 
+### Added
+
+- Added `tty::unix::from_fd()` to create a TTY from a pre-opened PTY's file-descriptors
```

Comment:
```suggestion
- `tty::unix::from_fd()` to create a TTY from a pre-opened PTY's file-descriptors
```

If we add `Added` to every entry in the Added section, we're not really gaining anything.